src/__tests__/Footer.test.js

```diff
@@ -0,0 +1,36 @@
+import { render, screen } from "@testing-library/react"
+import { BrowserRouter } from "react-router-dom"
+// import userEvent from "@testing-library/user-event"
+import Footer from "../components/Footer"
+
+describe("<Footer />", () => {
+  it("renders without crashing", () => {
+    render(
+      <BrowserRouter>
+        <Footer />
+      </BrowserRouter>
+    )
+  })
+  // it("renders a logo with a src and alt", () => {
+  //   const div = document.createElement("div")
+  //   render(
+  //     <BrowserRouter>
+  //       <Footer />
+  //     </BrowserRouter>,
+  //     div
+  //   )
+  // })
+  // it("has clickable links", () => {
+  //   render(
+  //     <BrowserRouter>
+  //       <Footer />
+  //     </BrowserRouter>
+  //   )
+  //   userEvent.click(screen.getByText("Meet the Cats"))
+  //   expect(screen.getByText("Meet the Cats")).toBeInTheDocument()
+  //   userEvent.click(screen.getByText("Add a New Cat"))
+  //   expect(screen.getByText("Add a New Cat")).toBeInTheDocument()
+  //   userEvent.click(screen.getByText("Adopt a Cat!"))
+  //   expect(screen.getByText("Adopt a Cat!")).toBeInTheDocument()
+  // })
```

Comment:
Right now this file is not testing anything.  Make sure to comment these back in and make them pass.